parsel/selector.py

```diff
@@ -341,6 +348,8 @@ def __init__(
         self,
         text: Optional[str] = None,
         type: Optional[str] = None,
+        body: Optional[bytes] = None,
+        encoding: Optional[str] = "utf8",
```

Comment:
This, and similar lines below, should probably match the corresponding lines from `create_root_node` above.

Reply:
@Gallaecio 
Applying   ` body: Optional[bytes]=b""` in `Selector.__init__`  as on `create_root_node` breaks argument typechecks on `__init__` and causing a lot of tests to fall (more than half) especially on tests with usual `text(str)` originated selectors.

Reply:
Please, have a look at my recent changes, and see if you are OK with them.

Reply:
@Gallaecio 
I am OK with them